public/views/index.html

```diff
@@ -10,7 +10,7 @@
     <meta name="viewport" content="width=device-width" />
     <meta name="theme-color" content="#000" />
 
-    <title>[[.AppTitle]]</title>
+    <title>Percona Monitoring and Management</title>
```

Comment:
There's also `[[.AppTitle]]` in `error.html`.

Also please add a comment above about the change in grafana code (useful when pulling changes from upstream):

```html
<!-- @PERCONA -->
<title>Percona Monitoring and Management</title>
```